src/simularium/VisGeometry/index.ts

```diff
@@ -276,6 +306,88 @@ class VisGeometry {
         fcam.addInput(this.camera, "position");
         fcam.addInput(this.controls, "target");
 
+        const grid1: ButtonGridApi = this.gui.addBlade({
```

Comment:
this could be reduced down to a `forEach` loop on an array of `[this.cam1, this.cam2, this.cam3]` so that there isn't so much repeated code in here.  Or `[{camera: this.cam1, label: "Cam 1"}, ...]`

Reply:
done!